.github/workflows/ci.yml

```
@@ -163,7 +163,7 @@ jobs:
           set -e
           cd examples/rust_with_cffi/
           python --version
-          pip install -U build cffi wheel
+          pip install -U build cffi 'setuptools>=70.1,<74'  # TODO: Remove pin once python-cffi/cffi#117 is solved
```

I think cffi patch is now released:

```suggestion
          pip install -U build cffi 'setuptools>=70.1'
```